wiki/Ranking_criteria/Metadata/en.md

```diff
@@ -265,4 +265,5 @@ This section covers what to do when romanising metadata yourself. If you use an
 ## Notes
 
 [^title-case]: Capitalise all major words and leave conjunctions such as `and`, `to`, `or` as well as articles like `the`, `a`, `an` lowercase.
-[^character-voice-actor]: `CV (Character Voice)` is used when song is sung in character by the character's voice actor. `VO (Voice Over)` is used when a song is sung in character by someone other than the character's voice actor.
+[^character-voice-actor]: `Character Voice (CV)` is used when song is sung in character by the character's voice actor. `Voice Over (VO)` is used when a song is sung in character by someone other than the character's voice actor.
```

Comment:
```suggestion
[^character-voice-actor]: `CV` (*character voice*) is used when song is sung in character by the character's voice actor. `VO` (*voice over*) is used when a song is sung in character by someone other than the character's voice actor.
```
"CV" and "VO" are the things that are actually written in metadata so that should be the only literal-styled part. the other words are just for explanation

Reply:
ppl thought this was a spelling error and anything that makes this clear that it just isnt works so i changed it to your variant lol